### PR TITLE
added qrcode for the premio wallet address and the issuer address. Al…

### PR DESCRIPTION
**Pull request by eoliveros**
…so link for the transactions for each wallet.

### CODE REVIEW

File: templates/dashboard.html [hunk 1]
```diff
@@ -14,6 +14,16 @@ <h5 class="card-title">{{ config["ASSET_NAME"] }} balance</h5>
             </div>
         </div>
 
+        <div class="card">
```
Comment (djpnewton) [inline templates/dashboard.html]: lets move this into the asset balance card above

File: templates/dashboard.html [hunk 2]
```diff
@@ -23,6 +33,24 @@ <h5 class="card-title">Master account WAVES balance</h5>
...
```
Comment (djpnewton) [inline templates/dashboard.html]: same here, merge this with the card above "Master account WAVES balance"

File: app_core.py [hunk 3]
```diff
@@ -14,8 +14,10 @@
 
 if os.getenv("TESTNET"):
     app.config["TESTNET"] = True
+    app.config["wavesexplorer"] = 'https://testnet.wavesexplorer.com'
```
Comment (djpnewton) [inline app_core.py]: make the config setting upper case like all the other ones

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: templates/dashboard.html (hunk 1 above)
Comment (eoliveros) [reply templates/dashboard.html]: moved.

File: templates/dashboard.html (hunk 2 above)
Comment (eoliveros) [reply templates/dashboard.html]: moved as well.

File: app_core.py (hunk 3 above)
Comment (eoliveros) [reply app_core.py]: wavesexplorer is now in CAPS.

File: templates/dashboard.html [hunk 4]
```diff
@@ -10,6 +10,8 @@
                 <h5 class="card-title">{{ config["ASSET_NAME"] }} balance</h5>
                 <p class="card-text">
                 {{ data["asset_balance"] }}
+		<center><a href="{{ data["wavesexplorer"] }}/address/{{ data["asset_address"]}}/tx">{{ data["premio_qrcode"] | safe }}</a><br/></center>
```
Comment (djpnewton) [inline templates/dashboard.html]: it looks like you are mixing tabs and spaces here, also just link to the address on the waves explorer (not the txs)

File: templates/dashboard.html [hunk 5]
```diff
@@ -10,6 +10,8 @@
...
+		<center><a href="{{ data["wavesexplorer"] }}/address/{{ data["asset_address"]}}/tx">{{ config["ASSET_NAME"] }} transactions</a><br/></center>
```
Comment (djpnewton) [inline templates/dashboard.html]: just link to the address and put the address in the link text

File: templates/dashboard.html [hunk 6]
```diff
@@ -19,10 +21,20 @@ <h5 class="card-title">{{ config["ASSET_NAME"] }} balance</h5>
                 <h5 class="card-title">Master account WAVES balance</h5>
                 <p class="card-text">
                 {{ data["master_waves_balance"] }}
+		<center><a href="{{ data["wavesexplorer"] }}/address/{{ data["master_waves_address"]}}/tx">{{ data["issuer_qrcode"] | safe }}</a><br/></center>
```
Comment (djpnewton) [inline templates/dashboard.html]: same as above

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: templates/dashboard.html (hunk 4 above)
Comment (eoliveros) [reply templates/dashboard.html]: im not sure i understand, you mean having the the url for the waves explorer? So it should look like this?:
```
		<center><a href="{{ data["wavesexplorer"] }}>{{ data["premio_qrcode"] | safe }}</a><br/></center>
```

Comment (djpnewton) [reply templates/dashboard.html]: no i mean remove the 'tx' at the end of the url

Comment (djpnewton) [reply templates/dashboard.html]: >address *on* the waves explorer

not address *of* the waves explorer

File: templates/dashboard.html [hunk 7]
```diff
@@ -10,6 +10,8 @@
                 <h5 class="card-title">{{ config["ASSET_NAME"] }} balance</h5>
                 <p class="card-text">
                 {{ data["asset_balance"] }}
+		<center><a href="{{ data["wavesexplorer"] }}/address/{{ data["asset_address"]}}">{{ data["premio_qrcode"] | safe }}</a><br/></center>
```
Comment (djpnewton) [inline templates/dashboard.html]: still mixed tabs and spaces here

File: templates/dashboard.html [hunk 8]
```diff
@@ -19,6 +21,8 @@ <h5 class="card-title">{{ config["ASSET_NAME"] }} balance</h5>
                 <h5 class="card-title">Master account WAVES balance</h5>
                 <p class="card-text">
                 {{ data["master_waves_balance"] }}
+		<center><a href="{{ data["wavesexplorer"] }}/address/{{ data["master_waves_address"]}}">{{ data["issuer_qrcode"] | safe }}</a><br/></center>
```
Comment (djpnewton) [inline templates/dashboard.html]: tabs mixed in here

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: templates/dashboard.html (hunk 7 above)
Comment (eoliveros) [reply templates/dashboard.html]: should be fixed

File: templates/dashboard.html (hunk 8 above)
Comment (eoliveros) [reply templates/dashboard.html]: should be fixed.

**Pull request merged**